src/api/app/components/request_decision_component.html.haml

```diff
@@ -30,12 +30,9 @@
       .mt-2
         - if @forward_allowed
           - @action.forward.each_with_index do |forward, index|
-            .form-check.m-2
+            .form-check.m-2.d-none
```

Comment:
I don't get it, you hide the checkbox but who will fill the `forward-...` parameters?

Reply:
Oh good catch! Fixing...

Reply:
So this will forward no matter what I click in the `RequestDecisionComponent` right?

Reply:
Right, that's bad, changing...